.github/workflows/ci.yml

```diff
@@ -93,16 +91,16 @@ jobs:
       if: ${{ matrix.python-version == '3.7' }}
       with:
         files: ./coverage.xml
-    - name: Integration test
-      env:
-        MONGODB_HOST: localhost
-        MONGODB_PORT: ${{ job.services.mongodb.ports[27017] }}
-        RADICAL_PILOT_DBURL:  mongodb://localhost:${{ job.services.mongodb.ports[27017] }}/test
-        TARGET_PATH: 'docs/source/getting_started.ipynb'
-      run: |
-        . testenv/bin/activate
-        jupyter nbconvert --clear-output --inplace $TARGET_PATH
-        jupyter nbconvert --to notebook --execute --inplace $TARGET_PATH
+    # - name: Integration test
+    #   env:
+    #     MONGODB_HOST: localhost
+    #     MONGODB_PORT: ${{ job.services.mongodb.ports[27017] }}
+    #     RADICAL_PILOT_DBURL:  mongodb://localhost:${{ job.services.mongodb.ports[27017] }}/test
+    #     TARGET_PATH: 'docs/source/getting_started.ipynb'
+    #   run: |
+    #     . testenv/bin/activate
+    #     jupyter nbconvert --clear-output --inplace $TARGET_PATH
+    #     jupyter nbconvert --to notebook --execute --inplace $TARGET_PATH
```

Comment:
this could be cleaned out?

Reply:
I would like to leave it there because once we stop having to test 3.6, using that code requires less resources than running a separate whole job. That is because an independent job needs to do all the setup we already do for this job. Would that work for you?

Reply:
I see, makes sense, ok!